src/components/input/LabeledDropdown.tsx

```diff
@@ -19,7 +20,6 @@ export function GenericLabeledDropdown<A> (props: LabeledDropdownProps<A>): Reac
               select
               helperText={error !== undefined ? 'Make a selection' : undefined}
               error={error !== undefined}
-              defaultValue=""
```

Comment:
State is defaulting to Alabama with these labeledDropdown changes

Reply:
I think it should default to empty

Reply:
Yep, got that thanks